IntroToCSharp/Shared/User/UserData.cs

```diff
@@ -27,17 +27,20 @@ public class UserStats
     /// The user's amount of gold.
     /// </summary>
     /// <value>Amount of gold</value>
-    public int Gold { get; private set; }
+    public int Gold { get => GoldAcquired - this.GoldSpent; }
 
+    public int GoldSpent {get; set;} //When the store 
+    public int GoldAcquired {get; private set;}
```

Comment:
@cloudthepain Just caught this one:
You will almost certainly need this to be a `{get; set;}` The database needs it to be public to be able to set the value when it loads.

Reply:
You just meant for GoldAcquired correct? If so, adjusted.

Reply:
Yes. Notice that `GoldSpent has a {get; set;}` this is because it is needed when the data is loaded in by the database.